samples/test-samples.cmake

```diff
@@ -69,6 +68,11 @@ foreach (SAMPLE ${SAMPLES_LIST})
     COMMAND ${CMAKE_COMMAND} --build ${SOURCE_DIR}/${SAMPLE}
     WORKING_DIRECTORY ${SAMPLE_BUILD_DIR})
 
+  # We do not provide attestation evidence for the host-verify program, so we build, but not run, the sample.
+  if (${SAMPLE} MATCHES host_verify)
+    continue()
+  endif ()
```

Comment:
Is it possible to just use some fake/old data and run it through the sample? It's best if people can run it easily

Reply:
It's a good idea, but since it's not possible to fake the expiration date of a certificate we will need to update it from time to time, which may cause some overhead.

Reply:
Hmm. Good point. I don't know enough about the evidence format to know what is feasible. Probably the right thing is to do as I suggested below and have a tool/code generate the evidence and then pass it to the sample. But if that tooling doesn't already exist we can open an issue to add it.